services/121-service/src/registration/validators/registrations-input-validator.ts

```diff
@@ -141,6 +141,10 @@ export class RegistrationsInputValidator {
       if (errorObj) {
         errors.push(errorObj);
       }
+
+      if (row.phoneNumber === undefined && program.allowEmptyPhoneNumber) {
+        row.phoneNumber = '';
+      }
```

Comment:
if (row.phoneNumber === undefined && program.allowEmptyPhoneNumber) {
        importRecord.phoneNumber = '';
      } else {
        importRecord.phoneNumber = row.phoneNumber;
      }

Reply:
I think this is a bit more readable. Otherwise good find :)

Reply:
@RubenGeo  This will not work, because you have to use row value in 
![image](https://github.com/global-121/121-platform/assets/57286575/3e19d8a4-768d-4716-922d-2f5f280f9d35)